### PR TITLE
chore(showcase): Adding ajonp as creator and site ajonp.com to showcase

### PR DESCRIPTION
**Pull request by ajonp**
## Description

I have added https://ajonp.com as a site showcase, as well as adding to the creators list.

Keep rocking Gatsby team! I can't wait to start using this site to walk through new demos.

If anyone has free time I would love to have a dev advocate live stream on our new meetup 😄 
https://link.ajonp.com/jamstackgr

### CODE REVIEW

**Comment by ajonp** [comment]
I am not familiar with Danger bot, but I think it needs to be updated to match
https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml

**Comment by lannonbr** [comment]
@ajonp So the file that you just linked is used for both the site showcase and starter library. If you scroll down to the 2nd section, you will be able to see the valid categories for the site showcase

File: docs/sites.yml [hunk 1]
```diff
@@ -7530,3 +7530,27 @@
...
+    - Blog
+    - CMS:sanity.io
+    - Firebase
+    - SEO
+    - Netlify
+    - Pagination
+    - RSS
+    - Search
+    - Styling:PostCSS
+    - Styling:Tailwind
```
Comment (pieh) [inline docs/sites.yml]: Whitelisted site categories are listed in https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml#L64-L117 , please remove those that are not in the list:
```
    - CMS:sanity.io
    - Firebase
    - Netlify
    - Pagination
    - RSS
	- Search
    - Styling:PostCSS
    - Styling:Tailwind
```

Comment (ajonp) [reply docs/sites.yml]: Cool thank you for clarification.

**Pull request merged**